scripts/linters/css_linter.py

```diff
@@ -87,36 +87,29 @@ def lint_css_files(self):
             raise Exception(
                 'ERROR    Please run start.sh first to install node-eslint '
                 'or node-stylelint and its dependencies.')
-        files_to_lint = self.all_filepaths
-        num_files_with_errors = 0
+
         failed = False
         stripped_error_messages = []
         full_error_messages = []
         name = 'Stylelint'
 
         stylelint_cmd_args = [
             node_path, stylelint_path, '--config=' + self.config_path]
-        result_list = []
-        for _, filepath in enumerate(files_to_lint):
```

Comment:
Good catch! These changes are reducing the lint check completion time by 10-15 mins. 

Can you please share a screenshot showing the linter report for lint issues in multiple files? [I'll approve the PR once you'll share the image of the linter report.]

Reply:
@Hudda @sajalasati We should audit other files to check whether we have such patterns in other places.

Reply:
> Good catch! These changes are reducing the lint check completion time by 10-15 mins.
> 
> Can you please share a screenshot showing the linter report for lint issues in multiple files? [I'll approve the PR once you'll share the image of the linter report.]

@DubeySandeep Thanks,
PTAL
![image](https://user-images.githubusercontent.com/55298925/110161744-cc2e5900-7e13-11eb-8212-afc396c23ba4.png)

Reply:
> @Hudda @sajalasati We should audit other files to check whether we have such patterns in other places.

@DubeySandeep I think it makes sense to do so. I've created #12117 now to track this and also added it to our list of tasks. Thanks for bringing into notice!